system/src/Grav/Common/Twig.php

```diff
@@ -183,6 +183,10 @@ public function processPage(Page $item, $content = null)
                 $twig_vars['content'] = $content;
                 $template = $item->template() . TEMPLATE_EXT;
                 $output = $local_twig->render($template, $twig_vars);
```

Comment:
I was just commenting the code; take a look the lines 185 and 189.Basically you're rendering output, setting template and rendering output again (discarding the old one).